README.md

```diff
@@ -34,17 +152,23 @@ docker exec --interactive --tty janis yarn run build-langs
 ```
 
 Note: if you're running the docker container built by serve-build.sh you'll have to update the container name from `janis` to `janis-build` in the above command.
-___
 
+### Static build scripts
 
-## Design assets
+Since we use React-Static as our framework to render our React components as a static progressive website, it's important for us to be able to test the final static build locally. In order to do this, we have two scripts.
 
-Current design mockups are stored in Google Drive as Sketch Files:
-https://drive.google.com/drive/u/1/folders/1Xg6739ixhIDrb7MdJAnO4_zCRu3kDmZQ
+**To build**
+```
+./scripts/build.sh
+```
 
----
+**To build and server**
```

Comment:
**To build and serve**  - r

Reply:
good point, we made the build script before the build and serve and I haven't used it since.